* What does the following error message tell us? *

ArgumentError: wrong number of arguments (1 for 2)
  from (irb):1:in `calculate_product'
  from (irb):4
  from /Users/username/.rvm/rubies/ruby-2.0.0-p353/bin/irb:12:in `<main>'

This error message tells us 'calculate_product' takes 2 parameters, but when it was called,
    only 1 argument was provided.